setup.cfg

```diff
@@ -4,9 +4,14 @@ exclude=webapp-django/crashstats/*/migrations/*,docs/conf.py,socorro/unittest/te
 max-line-length = 100
 
 [tool:pytest]
-addopts = -rsxX --tb=native
+# -rsxX        - show skipped, failed, and passed tests
+# --tb=native  - print native traceback
+# -p no:django - disable the pytest-django plugin for Socorro tests
+addopts = -rsxX --tb=native -p no:django
 norecursedirs = .git docs scripts config docker __pycache__
 testpaths = socorro/unittest/
+# NOTE(willkg): These values are used by Travis. Once we ditch Travis, we can
+# probably either ditch these values or fix them or both.
```

Comment:
Figured I'd add this note so that future me doesn't spend 10 minutes trying to figure out why these values are the way they are like present me just did because past me didn't think to tell the rest of us what's going on.